plugins/modules/pacemaker_stonith.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2025, Dexter Le <dextersydney2001@gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+module: pacemaker_stonith
+short_description: Manage pacemaker STONITH
+author:
+  - Dexter Le (@munchtoast)
+version_added: 11.2.0
+description:
+  - This module manages STONITH in a Pacemaker cluster using the pacemaker CLI.
```

Comment:
It's either
```suggestion
  - This module manages STONITH in a Pacemaker cluster using the Pacemaker CLI.
```
or
```suggestion
  - This module manages STONITH in a Pacemaker cluster using the C(pacemaker) CLI.
```
if you're refering to the CLI tool of that name.